python/rikai/types/geometry.py

```diff
@@ -102,6 +111,33 @@ class Box2d(ToNumpy, Sequence, ToDict, Drawable):
     >>> draw.rectangle(box, fill="green", width=2)
     """
 
+    def left_top(self):
+        return self.xmin, self.ymin
+
+    def aligned(
+        self, base_drawable: ComposableDrawable, algorithm: str = "left-top"
+    ):
+        (new_xmin, new_ymin) = base_drawable.left_top()
+        return Box2d(
+            new_xmin,
+            new_ymin,
+            self.xmax - self.xmin + new_xmin,
+            self.ymax - self.ymin + new_ymin,
+        )
+
+    def with_label(
```

Comment:
cal this take more text customization, i.e., `font size,  background color` and et.

Reply:
If we add more parameters here, then we can't make it a synonym of `+`, maybe we should consider remove the symbol version of this method.